vulcanai2/plotters/visualization.py

```diff
@@ -201,8 +249,13 @@ def display_confusion_matrix(cm, class_list=None):
     divider = make_axes_locatable(ax)
     cax = divider.append_axes("right", size="4%", pad=0.03)
     plt.colorbar(im, cax=cax)
-    plt.show(False)
 
+    if save_path:
+        time = get_time()
+        save_path = save_path + '/confusion_matrix' + '_' + time + '.png'
```

Comment:
you could have a "get_save_path" function that replaces the get_time function, in case we or someone decided they wanted to change the standard save path later on